core/domain/base_model_validators.py

```diff
@@ -97,9 +98,17 @@ def __init__(self, field_name, model_class, model_ids):
                 'When fetching instances of UserSettingsModel, please use ' +
                 'UserSettingsModelFetcherDetails instead of ' +
                 'ExternalModelFetcherDetails')
+        filtered_model_ids = []
+        for model_id in model_ids:
+            if not model_id:
```

Comment:
Is this check for empty string or None or both?

Reply:
It is for both None and empty string.